### ISSUE TITLE
invalid option in `format' on 'ruRU' locale

### ISSUE THREAD

**Issue by pshurgal** [Hoizame/AtlasLootClassic#234]
**Lua error**
```
invalid option in `format' 
```

**Describe the bug**
Lua error appears on log in

**To Reproduce**
Just log into the game

**Expected behavior**
There is no error

**Screenshots**
![image](https://user-images.githubusercontent.com/244044/189653638-f7c8408d-bd3c-4e42-99ff-bcc8838b414d.png)

**Additional context**
```
Date: 2022-09-12 15:22:34
ID: 1
Error occured in: Global
Count: 1
Message: ...Ons\AtlasLootClassic_DungeonsAndRaids\data-wrath.lua line 5058:
   invalid option in `format'
Debug:
   [string "=[C]"]: format()
   [string "@AtlasLootClassic_DungeonsAndRaids\data-wrath.lua"]:5058: in main chunk
   [string "=[C]"]: LoadAddOn()
   [string "@AtlasLootClassic\Loader.lua"]:202: LoadModule()
   [string "@AtlasLootClassic\Data\AutoSelect.lua"]:1083: RefreshOptions()
   [string "@AtlasLootClassic\Loader.lua"]:139: func()
   [string "@AtlasLootClassic\AtlasLoot.lua"]:42:
      AtlasLootClassic\AtlasLoot.lua:34
Locals:
(*temporary) = "%Попыток осталось"
(*temporary) = "1-24"

AddOns:
  Swatter, v3.4.6817 (SwimmingSeadragon)
  WowheadLooter, v30400
  Atlas, vv1.51.06
  AtlasLootClassic, vv3.0.1
  AtlasLootClassicDungeonsAndRaids, vv3.0.1
  BlizRuntimeLib_ruRU v3.4.0.30400 <none>
  (ck=c6)
```

**Comment by pshurgal** [comment]
Seems line localization for 'ruRU' locale is incorrect and misses formatting specifier:
`AtlasLootClassic_DungeonsAndRaids\Locales\constants.ru.lua:490`
```
AL["%s Attempts left"] = "%Попыток осталось"
```

**Comment by Hoizame** [comment]
Its fixed for next version with #233 :)
I will release it in the next minutes.
